### ISSUE TITLE
Open URLS in a new tab by default

### ISSUE THREAD

**Issue by abishekk92** [Firehed/firstworldproblems.biz#9]
Open the URLs in a new tab by default than the current redirects. It makes for a poor UX when you have to go back all the time to the parent URL to checkout the rest.

**Comment by ReefPig** [comment]
Yeah the UX is awful! Submit a pull request and we will merge it in!

**Comment by abishekk92** [comment]
Sure will do.
